client/js/store/modules/game.js

```diff
@@ -71,6 +72,33 @@ const initialState = resetState();
 export default {
   state: initialState,
   getters: {
+    // TODO: Replace this with an actual deck array :)
+    cards(state, getters) {
+      if (!getters.player || !getters.opponent) {
+        return [];
+      }
+      return [
+        // Global cards
+        ...state.deck,
+        ...state.scrap,
+        // Player cards
+        ...getters.player.hand,
+        ...getters.player.points,
+        ...getters.player.faceCards,
+        // Opponent cards
+        ...getters.opponent.hand,
+        ...getters.opponent.points,
+        ...getters.opponent.faceCards,
+      ]
+        .map(({ id, rank, suit, name, ruleText }) => ({
+          id,
+          rank,
+          suit,
+          name,
+          ruleText,
+        }))
+        .sort(compareByRankThenSuit);
```

Comment:
@itsalaidbacklife this is what I was referring to when I said I needed a full set of cards in order to get the information I need to show an overlay. I'm still thinking through the all the things this touches but the idea is to use this array instead of the players hand when looking at a card that is ouside of it (field, royals, opponents hand, etc).

There also need to be some other assumptions like:
- Can't view overlay on cards in the scrap (for now)
- What to do with the click events for cards that already have them
    - `@click="targetOpponentPointCard(index)"`
    - `@click="targetOpponentFaceCard(-index - 1)"`
    - `@click="targetOpponentFaceCard(index)"`
- Logic changes from changing from an index to the need to do a lookup by id or similar